docker-nlp-toolset/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM maluuba/tomcat7
+MAINTAINER Maluuba NLP Team <nlp@maluuba.com>
+
+RUN apt-get install -qqy git docker.io
```

Comment:
Why are you installing docker inside the container? Is the toolset going to be running docker as well?